drivers/auxiliary/wandererbox_pro_v3.h

```diff
@@ -0,0 +1,280 @@
+        double DHTTread = 0;
+        //Total Current
+        double Tcurrentread = 0;
+        //19V Current
+        double V19currentread = 0;
+        //Adjustable Regulator Current
+        double ARcurrentread = 0;
+        //Power Monitor
+        void updatePower(double Tcurrent,double V19_current,double AR_current,double voltage);
+        double voltageread = 0;
+        //USB31
+        void updateUSB31(int value);
+        int USB31read = 0;
+        //USB32
+        void updateUSB32(int value);
+        int USB32read = 0;
+        //USB33
+        void updateUSB33(int value);
+        int USB33read = 0;
+        //USB21
+        void updateUSB21(int value);
+        int USB21read = 0;
+        //USB22
+        void updateUSB22(int value);
+        int USB22read = 0;
+        //DC34
+        void updateDC34(int value);
+        int DC34read = 0;
+        //DC5
+        void updateDC5(int value);
+        int DC5read = 0;
+        //DC6
+        void updateDC6(int value);
+        int DC6read = 0;
+        //DC7
+        void updateDC7(int value);
+        int DC7read = 0;
+        //DC8_9
+        void updateDC8_9(int value);
+        int DC8_9read = 0;
+        //DC10_11
+        void updateDC10_11(int value);
+        int DC10_11read = 0;
+        //DC34set
+        void updateDC34SET(double value);
+        int DC34SETread = 0;
+        
+        bool setDewPWM(uint8_t id, uint8_t value);
+
+        //DC Control//////////////////////////////////////////////////////////////////////////////////
+        ISwitchVectorProperty dc3_4ControlSP;
+        ISwitch dc3_4ControlS[2];
+        ISwitchVectorProperty dc8_9ControlSP;
+        ISwitch dc8_9ControlS[2];
+        ISwitchVectorProperty dc10_11ControlSP;
+        ISwitch dc10_11ControlS[2];
+        //USB Control//////////////////////////////////////////////////////////////////////////////////
+        ISwitchVectorProperty usb31ControlSP;
+        ISwitch usb31ControlS[2];
+        ISwitchVectorProperty usb32ControlSP;
+        ISwitch usb32ControlS[2];
+        ISwitchVectorProperty usb33ControlSP;
+        ISwitch usb33ControlS[2];
+        ISwitchVectorProperty usb21ControlSP;
+        ISwitch usb21ControlS[2];
+        ISwitchVectorProperty usb22ControlSP;
+        ISwitch usb22ControlS[2];
+        //DC5 Control////////////////////////////////////////////////////////////////
+        INumber dc5ControlN[1];
+        INumberVectorProperty dc5ControlNP;
+        enum
+        {
+            DC5,
+        };
+        //DC6 Control////////////////////////////////////////////////////////////////
+        INumber dc6ControlN[1];
+        INumberVectorProperty dc6ControlNP;
+        enum
+        {
+            DC6,
+        };
+        //DC7 Control////////////////////////////////////////////////////////////////
+        INumber dc7ControlN[1];
+        INumberVectorProperty dc7ControlNP;
+        enum
+        {
+            DC7,
+        };
```

Comment:
All INumberVectorProperty should be using INDI::PropertyNumber

Reply:
Ok